.github/workflows/publish.yml

```diff
@@ -0,0 +1,49 @@
+name: Package and Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  PyPi-Deploy:
+    name: Publish Python Package to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.x'
```

Comment:
It will only work with Python >= 10. Please test for 3.10, 3.11, 3.12.

Reply:
By using strategy matrix, we will have 3 runners running in with different python versions and it will be publish the package 3 times with these python versions. Is this what is needed here?

Reply:
Done.

Reply:
@forman do we also need to add testing the js library for these node-version: [16.x, 18.x, 20.x]?

Reply:
Yes.

Reply:
Done